smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -18,13 +18,17 @@ open class HttpProtocolServiceClient(
     private val serviceName: String = ctx.settings.sdkId
 
     fun render(serviceSymbol: Symbol) {
```

Comment:
Refactoring, plus encoder / decoder fields are only added to clients when needed.